test/integration/ink_editor_spec.mjs

```diff
@@ -941,7 +944,6 @@ describe("Ink Editor", () => {
 
           await page.waitForSelector(getEditorSelector(1));
           await waitForSerialized(page, 1);
-          await page.waitForSelector(getEditorSelector(1));
```

Comment:
Note that we already wait for this selector just above this line, so this duplicate definition can simply be removed.